init.lua

```diff
@@ -55,7 +66,6 @@ if ts_furniture.enable_sitting then
 	end
 
 	ts_furniture.stand = function(player, name)
-		player:set_eye_offset({x = 0, y = 0, z = 0}, {x = 0, y = 0, z = 0})
```

Comment:
you removed this for being unncessary becaus there's no other `set_eye_offset` call anywhere else, correct?

Reply:
I removed both set_eye_offset. The engine positions the eyes correctly without those functions. With them it moved the eyes below the seat. I assume the engine didn't handle it correct in the past and this "workaround" became a bug.

Reply:
After your changes the eyes are positioned as if you were standing on the bench or chair, so this is worse than before.

Reply:
This is not true at all.

Picture of sitting on bench
![screenshot_20240428_121600](https://github.com/minetest-mods/ts_furniture/assets/67873653/0262636c-006e-4ca1-bdf0-acf493992ccf)
Picture of standing in same position and almost the same angle
![screenshot_20240428_121619](https://github.com/minetest-mods/ts_furniture/assets/67873653/001d5446-f71b-405d-875a-08a3a21984fa)
Picture of sitting in first person with mod before my patch: (It's some wood from more trees that's why it doesn't use texture from texture pack. It's from this mod.)
![screenshot_20240428_121828](https://github.com/minetest-mods/ts_furniture/assets/67873653/4162e7c3-5e32-4e61-b840-cb1dfd3700d1)

Reply:
As you can see in third person the difference at where the eyes are is not much when sitting or standing. However technically when sitting the player position is on top of bench or chair, so it means it's working perfectly because the eyes are where they are supposed to be.

Reply:
master, standing:
![screenshot_20240428_124843](https://github.com/minetest-mods/ts_furniture/assets/1042418/5bf93c5b-514c-4b34-95c1-e8b0c15f11d5)
master, sitting:
![screenshot_20240428_124851](https://github.com/minetest-mods/ts_furniture/assets/1042418/f4b8e77b-900b-4dea-9090-f3c21ffdf9c7)
PR, sitting:
![screenshot_20240428_125155](https://github.com/minetest-mods/ts_furniture/assets/1042418/5632f1b6-6df1-47a4-8aa9-48e572480cf0)

the eye position is even higher than when standing. makes no sense.

Reply:
It doesn't look like that on 5.8 or 5.7. Instead the camera is in the seat. And the camera should not be in the seat.
Is that a regression in 5.9 or what's supposed to happen? I think the default behavior in 5.8 isn't bad. Maybe just fix the dev version to behave like 5.8 and 5.7?

Reply:
![grafik](https://github.com/minetest-mods/ts_furniture/assets/67873653/b805aeb8-b1aa-4082-bbc3-3cfac4e5f1ef)
This is what sitting on the chair looks like on master in Minetest 5.8 which is the current release everyone is playing.

Reply:
I can confirm that the camera offset is indeed broken on 5.8.
At this point this issue goes beyond the scope of this PR and someone will need to figure out where exactly this goes wrong and file an issue on the engine or MTG (it won't be me).

Reply:
It's already been like this in 5.7 and possibly earlier. Maybe 5.9 should be considered broken AND YOU SHOULD MERGE.